blog/2025-05-23-distr-docs/index.mdx

```diff
@@ -0,0 +1,213 @@
+
+Docusaurus uses Infima ([infima.dev](https://infima.dev/)) at its core.
+It is actually pretty tightly coupled, so if you want to style specific pages or use CSS classes to style some components, you need to get familiar with Infima.
+Sadly, Infima itself isn't that mature yet and misses a lot of components and possibilities that frameworks like Bootstrap or Tailwind are offering.
+
+Also, integrating Bootstrap or Tailwind is not really possible.
+
+Starlight and Astro, on the other hand, don't come with a specific CSS framework built in, but provide a one-line command with their CLI to integrate Tailwind.
+
+Read more about [Starlight and Tailwind](https://starlight.astro.build/guides/css-and-tailwind/).
+
+:::note
+It took a little until Starlight fully supported Tailwind v4.
+But once [#2862](https://github.com/withastro/starlight/issues/2862) was completed,
+upgrading Starlight from Tailwind v3 to v4 worked very well and was pretty well documented.
+:::
+
+### Docusaurus vs. Starlight: SEO
+
+Both frameworks provide all the necessary features to configure the needed meta data tags out of the box.
+You can read the [SEO guide for Docusaurus](https://docusaurus.io/docs/seo) or check out all the
+[SEO Integrations for Astro](https://astro.build/integrations/?search=&categories%5B%5D=performance%2Bseo)
+which come preconfigured with Starlight.
+
+As both frameworks support static rendering at build time, page speed is lightning fast and is only slowed down
+by external libraries that are integrated.
+
+| Feature                          | Docusaurus                  | Starlight |
+| -------------------------------- | --------------------------- | --------- |
+| Sitemap generation               | ✅                          | ✅        |
+| Metadata & Open Graph            | ✅                          | ✅        |
+| Broken Link Checker              | ✅                          | ✅        |
+| Image Optimization & compression | ⚠️ (hard to get it working) | ✅        |
+
+It should be noted that most features for Starlight are available via plugins.
+
+:::tip
+It is generally recommended to run your documentation through Google's [PageSpeed Insights Test](https://pagespeed.web.dev/)
+:::
+
+### Docusaurus vs. Starlight: Developer experience
+
+A framework for writing documentation should not hold you back from writing documentation quickly:
+
+- It should not be a lot of effort to keep up to date.
+- It should be fast, blazing fast.
+- It should have great integrations, documentation, and community support.
+- It should feel modern and fun to use.
+
+#### Maintenance work
+
+Docusaurus' hard dependency on React introduces a natural update hurdle.
+Although React, Docusaurus, and Infima are all Open Source projects from Meta (formerly Facebook), they are developed by different teams.
+Not only does the Docusaurus team test compatibility with the latest React version, but also all plugins need to ensure compatibility.
```

Comment:
I'm not sure what this means. Is it a good thing if they test the latest version? It should be but you make it sound like it's not…